_posts/photography/2015-03-10-my_gear.markdown

```diff
@@ -0,0 +1,136 @@
+
+## **Contax G1** 
+with **Contax G Zeiss 28mm f/2.8 Biogon**
+
+This is a new arrival to fill the gap of a small quality 35mm camera that I will carry with me as a backup to the medium format. 
+
+I used to have an Olumpus XA2 for this job but the constraints of this camera led me to search for a better one. 
+
+I found the Contax G1 in a bargain and bought it, I don't need to say much about the Contax G system, it is one of the best 35mm cameras you can get. 
+[Ken Rockwell Review](http://www.kenrockwell.com/contax/g-system.htm)
+
+The only problem is that eventually it is not going to be a replacement of the Olympus XA2. Its bigger and more fragile, I carry the XA2 everywhere inside my everyday backpack, something you can't do with the precious Contax. 
+
+Where I use the Contax: 
+
+- Backup to the Bronica GS-1 in my photography roadtrips
+- Main camera in my urban wanderings
+- Loaded with black and white film for my black and white projects :) 
+
+## **Olympus OM2n** 
+with **Zuiko Auto-W 28mm f/3.5**
+
+This is a very good camera but I have never used it. Bought it the same time I bought the Olympus XA2 and maybe thats a good excuse for not using it. 
+
+The camera is very small, has a solid build and a fantastic viewfinder. I can not say much about it because I have shoot only 3 or 4 films with this one. 
+
+The logic says that now with the entry of the contax G1 this camera became obsolete, but I don't think so, the Contax is an AF rangefinder with expensive lenses and accessories where the OM2n is a classic manual focus SLR with a huge selection of good and affordable lenses. 
+
+I don't know if I ever shoot actively with it but I will not sell it for the following 3 reasons:
+
+1. It is my only 35mm SLR camera
+2. It is not worth a lot of money
+3. It is one of the best looking SLRs 
+
+## **Olympus XA2**
+
+This one is one of my favorites cameras. Why? Because I have it with me all the time! 
+
+XA2 comes in a tiny clamshell body with an awesome, sharp, contrasty little Zuiko 35mm 1:3.5 4-element lens.
+
+XA2 offers automatic exposure and zone focusing. This means that the only thing you can control is the focus zones which are splited in three groups: close (1.2m-1.8m), near (1.2-6.3m) and far (6.3m-infinity).
+
+This focusing system is one of the biggest disadvantages in my opinion, I have many bad focused images and with a mediocre quality this images become unusable. 
+
+Overall this is my favorite take-everywhere film camera and responsible for this great set: [Roadtrip Snapshots](http://rpk.io/photo_essays/roadtrip_snapshots/)
+
+## **Olympus OMD EM-5**
+with **M.Zuiko 12-50mm f/3.5-6.3 ED** and **Panasonic LUMIX G 20mm f/1.7**
+
+Its 2015 you can not leave in this year and not have a good digital camera. There is moments where the advantages of a digital camera make it indispensable.
```

Comment:
It's 2015 you can not leave in this year and not have a good digital camera. There are moments where the advantages of a digital camera make it indispensable.